channeldb/kvdb/etcd/commit_queue_test.go

```diff
@@ -7,24 +7,25 @@ import (
 	"sync"
 	"sync/atomic"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
 // TestCommitQueue tests that non-conflicting transactions commit concurrently,
 // while conflicting transactions are queued up.
 func TestCommitQueue(t *testing.T) {
-	// The duration of each commit.
-	const commitDuration = time.Millisecond * 500
-	const numCommits = 4
-
+	const numCommits = 3
 	var wg sync.WaitGroup
 	commits := make([]string, numCommits)
 	idx := int32(-1)
 
-	commit := func(tag string, sleep bool) func() {
+	commit := func(tag string, commit chan struct{},
+		block *sync.Mutex) func() {
+
 		return func() {
+			if commit != nil {
```

Comment:
can names this `signalStart` or something other that is more descriptive than `commit`